src/css/controls/flags/cast.less

```diff
@@ -9,9 +9,10 @@
         display: none;
     }
 
-    .jw-icon-audio-tracks,
     .jw-icon-hd,
-    .jw-icon-fullscreen {
+    .jw-captions,
```

Comment:
@karimJWP this will hide captions rendered by the player. Have you confirmed that browser-rendered captions don't display when we're casting?